components/Header/index.js

```diff
@@ -0,0 +1,93 @@
+import { useTransactionsContext } from "@/utils/TransactionsContext/TransactionsContext";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+
+export default function Header() {
+    const router = useRouter();
+    const activeNavPoint = router.pathname;
+    const { handleFilterChange } = useTransactionsContext();
+
+    const handleNavClick = (path) => {
+        handleFilterChange({ type: null, pattern: null })
+    };
+
+    return (
+        <StyledHeader>
+            <LogoContainer>
+                <StyledEmoji aria-hidden="true">💸</StyledEmoji>
+            </LogoContainer>
+            <HomeLink
+                href="/"
+                aria-label="Zurück zur Startseite von mon-ey"
+                aria-current={activeNavPoint}
+                onClick={() => handleNavClick()}
+            >
+                <AppName>
+                    mon-<span className="rotateEy">ey</span>
```

Comment:
Please don't make use of 'className' since you use styled-components in this project.